test/extended/cli/explain.go

```diff
@@ -411,18 +411,23 @@ var (
 				pattern: `DESCRIPTION\:.*`,
 			},
 			{
-				gv:      schema.GroupVersion{Group: "console.openshift.io", Version: "v1alpha1"},
+				gv:      schema.GroupVersion{Group: "console.openshift.io", Version: "v1"},
```

Comment:
Updated since the ConsolePlugin uses v1 already some time, see https://github.com/openshift/api/pull/1186, fyi @jhadvig